dcgm/metadata.csv

```diff
@@ -0,0 +1,24 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+dcgm.dec_utilization,gauge,,percent,,Decoder utilization (in %).,0,dcgm,,
+dcgm.enc_utilization,gauge,,percent,,Encoder utilization (in %).,0,dcgm,,
+dcgm.fb_free,gauge,,mebibyte,,Framebuffer memory free (in MiB).,0,dcgm,,
+dcgm.fb_used,gauge,,mebibyte,,Framebuffer memory used (in MiB).,0,dcgm,,
+dcgm.temperature,gauge,,degree celsius,,GPU temperature (in C).,0,dcgm,,
+dcgm.gpu_utilization,gauge,,percent,,GPU utilization (in %).,0,dcgm,,
+dcgm.mem.clock,gauge,,megahertz,,Memory clock frequency (in MHz).,0,dcgm,,
+dcgm.mem.copy_utilization,gauge,,percent,,Memory utilization (in %).,0,dcgm,,
+dcgm.nvlink_bandwidth.count,count,,,,Total number of NVLink bandwidth counters for all lanes,0,dcgm,,
+dcgm.pcie_replay.count,count,,,,Total number of PCIe retries.,0,dcgm,,
+dcgm.power_usage,gauge,,watt,,Power draw (in W).,0,dcgm,,
+dcgm.sm_clock,gauge,,megahertz,,SM clock frequency (in MHz).,0,dcgm,,
+dcgm.total_energy_consumption.count,count,,,,Total energy consumption since boot (in mJ).,0,dcgm,,
```

Comment:
Not a blocker but maybe we could start the process to add this new unit and update the metadata file once that's done? Link to the docs to do that in the Implementation card IIRC

Reply:
On it!